cmake/catkin_package.cmake

```diff
@@ -374,6 +377,15 @@ function(_catkin_package)
       endif()
     endforeach()
 
+    set(PKG_COMPILE_OPTIONS ${${PROJECT_NAME}_COMPILE_OPTIONS})
+    separate_arguments(PKG_COMPILE_OPTIONS)
```

Comment:
I would expect the arguments to be passed separately and not in a single string, e.g.:

`set(${PROJECT_NAME}_COMPILE_OPTIONS "-Wall" "-DBla" "-fomit-frame-pointer")`

And while setting a global variable like this is possible it is hard to discover. Having a documented argument in `catkin_package` is imo preferred.